tests/seed.js

```diff
@@ -24,4 +26,24 @@ const populateUsers = done => { //eslint-disable-line
     }).then(() => done()); //eslint-disable-line
 };
 
-module.exports = {users, populateUsers};
\ No newline at end of file
+const setUuid = (index) => {
+    return new Promise((resolve,reject) => {
```

Comment:
remove this function and get uuid from the created user

Reply:
Removed